sw/device/tests/chip_power_idle_load.c

```diff
@@ -195,15 +195,17 @@ bool test_main(void) {
       .ping_timeout = UINT16_MAX,
   };
 
-  // 5. Configure Alerts
-  CHECK_STATUS_OK(alert_handler_testutils_configure_all(&alert_handler, config,
-                                                        kDifToggleEnabled));
+  // 5. Configure Alerts as unlocked so pings are disabled.
+  // TODO(lowrisc/opentitan#23644) enable locked when fixed to enable pings.
+  CHECK_STATUS_OK(
+      alert_handler_testutils_configure_all(&alert_handler, config,
```

Comment:
We still want to lock the configuration but avoid setting the ping mechanism. Unfortunately, this function doesn't support this. 

You will probably have to create a local implementation of the function to achieve the expected behavior. 

The other option is to refactor the function, but that is a bigger task.

Reply:
I think it is not strictly necessary to lock them for this test, since locking disabled only blocks outgoing pings. In any case, https://github.com/lowRISC/opentitan/issues/23644 is an M5 bug, and I intend to fix it.

Reply:
SG. Not locking the alert means that the config is not applied. I agree that this change makes sense since you intend to adress #23644.